src/validation.h

```diff
@@ -107,6 +107,8 @@ static const int64_t BLOCK_DOWNLOAD_TIMEOUT_PER_PEER = 500000;
 static const int64_t DEFAULT_MAX_TIP_AGE = 24 * 60 * 60;
 /** Maximum age of our tip in seconds for us to be considered current for fee estimation */
 static const int64_t MAX_FEE_ESTIMATION_TIP_AGE = 3 * 60 * 60;
+/** Maximum block age in seconds for us to be considered suitable for fee estimator */
+static const int64_t MAX_FEE_ESTIMATOR_BLOCK_AGE = 48 * 60 * 60;
```

Comment:
Should explain where this magic number comes from and why it is a safe choice considering the other constants in policy/fees.h. Ideally you'd express it using existing constants

Reply:
Na Magic here, just picked 2 days .. I think it should be a value that would produce similar fee rates right?

Reply:
2 days is still magic ;) "for us to be considered suitable" also sounds confusing to me, could you reword this?

Reply:
I'll change to a reasonable value, meanwhile I'm waiting for concept ACK.